src/components/PaymentLog.vue

```diff
@@ -1391,12 +1391,26 @@ export default {
         const { total, actualDownpayment, rePayment } =
           this.salesLogForm?.repayment_duration_id?.name === "six_months" && !this.FixedRepayment && this.salesLogForm?.business_type_id?.slug.includes('ap')
             ? decliningRepaymentCalculator(
+              this.selectedProduct.price,
+              data0,
+              data,
+              this.selected_discount?.percentage_discount
+            )
+            : data0.business_type_id.slug.includes("cash_loan") ||
```

Comment:
Cant we do something like this ["cash_loan", "ap_rentals", "ap_super", "etc"].includes(data0.business_type_id.slug)?